pagination/keysetpagination/paginator.go

```diff
@@ -60,6 +60,13 @@ func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 	var item I
 	id := (&pop.Model{Value: item}).IDField()
 	return func(q *pop.Query) *pop.Query {
+		if q.Connection.Dialect.Name() == "mysql" {
+			return q.
+				Limit(p.Size()+1).
+				Where(fmt.Sprintf("`%s` > ?", id), p.Token()).
+				Order(fmt.Sprintf("`%s` ASC", id))
```

Comment:
I looked at pop, you should be able to use `q.Connection.Dialect.Quote()` which will quote the string according to the database being used, making sure this works on every dialect!